snuba/datasets/storages/profiles.py

```diff
@@ -0,0 +1,92 @@
+from snuba.clickhouse.columns import UUID, ColumnSet, DateTime
+from snuba.clickhouse.columns import SchemaModifiers as Modifiers
+from snuba.clickhouse.columns import String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.profiles_processor import ProfilesMessageProcessor
+from snuba.datasets.schemas.tables import TableSchema, WritableTableSchema
+from snuba.datasets.storage import ReadableTableStorage, WritableTableStorage
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.table_storage import build_kafka_stream_loader_from_settings
+from snuba.query.processors.conditions_enforcer import OrgIdEnforcer, ProjectIdEnforcer
+from snuba.query.processors.table_rate_limit import TableRateLimit
+from snuba.query.processors.type_converters.uuid_column_processor import (
+    UUIDColumnProcessor,
+)
+from snuba.query.processors.uniq_in_select_and_having import (
+    UniqInSelectAndHavingProcessor,
+)
+from snuba.utils.streams.topics import Topic
+
+PROFILES_LOCAL_TABLE_NAME = "profiles_local"
+PROFILES_DIST_TABLE_NAME = "profiles_dist"
+
+
+processors = [
+    UniqInSelectAndHavingProcessor(),
```

Comment:
This processor was a workaround for a ClickHouse 20.3 bug and will be deprecated. I think you probably don't need it here.